frontend/spec/features/products_spec.rb

```diff
@@ -9,11 +9,8 @@
     ((first_store = Spree::Store.first) && first_store.name).to_s
   end
 
-  before(:each) do
```

Comment:
This should do exactly the same as manually visiting the path into each test. It is not happening once for all tests (that's `before(:all)` if I recall correctly).